sdks/java/io/hadoop-format/src/test/java/org/apache/beam/sdk/io/hadoop/format/HadoopFormatIOSequenceFileTest.java

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
do we still need HadoopOutputFormatIOIT? This one seems to be a full replacement, with more functionality.